domain/consensus/processes/consensusstatemanager/resolve_block_status_test.go

```diff
@@ -261,7 +261,12 @@ func TestTransactionAcceptance(t *testing.T) {
 			t.Fatalf("Error creating tip: %+v", err)
 		}
 
-		finalTipHash, _, err := testConsensus.AddBlock([]*externalapi.DomainHash{finalTipSelectedParentHash, redHash, blueHash}, nil,
+		finalTipScriptPublicKey := &externalapi.ScriptPublicKey{Script: []byte{4}, Version: 0}
```

Comment:
What changed?

Reply:
The red block reward is bestowed to the merging block instead of disappearing